src/color.rs

```diff
@@ -1,7 +1,7 @@
 /// A terminal color.
 pub trait Color {
-    /// Convert this to its ANSI value.
-    fn to_ansi_val(self) -> u8;
+    /// Get an escape code corresponding to the color value.
+    fn to_escape_code(self, is_background: bool) -> String;
```

Comment:
I thought you might, I fail to see how it's a big deal since it's not something that's done that often, but implementing `Display` sounds like a better way to do it altogether, completely agree.

I'll close this then.